Bindings/Java/OpenSimJNI/Test/testContext.cpp

```diff
@@ -176,6 +176,14 @@ int main()
     assert(context->getLocked(dr_elbow_flexNew));
     ASSERT_EQUAL(0.5, context->getValue(dr_elbow_flexNew), 0.000001);
 
+    // Exercise Editing workflow
+    OpenSim::Body& bdy = model->updBodySet().get("r_humerus");
+    AbstractProperty& massProp = bdy.updPropertyByName("mass");
```

Comment:
Indeed that's what the GUI executes when editing from the "Properties" window. I'll add a note.